lcd.cpp

```diff
@@ -958,6 +1036,13 @@ void LCD_setup() {
   pinMode(BEEPER, OUTPUT);
   digitalWrite(BEEPER, LOW);
 
+#ifdef USING_12864_WITH_ENCODER_INTERRUPTS
+  // set original pins as inputs, but do not enable their pull-ups
+  // new pins that are interruptable will enable pull-ups
+  pinMode(BTN_EN1_OLD, INPUT);
```

Comment:
These are defined, connections are soldered (as per previous code comment) and then never used?

Reply:
They will remain connected because of the ramps 1.4 shield connection, so they need to be configured as inputs.  But they aren't used otherwise.